shell/platform/android/external_view_embedder/external_view_embedder.cc

```diff
@@ -66,22 +66,22 @@ SkCanvas* AndroidExternalViewEmbedder::GetRootCanvas() {
   return nullptr;
 }
 
+void AndroidExternalViewEmbedder::Reset() {
+  composition_order_.clear();
+  picture_recorders_.clear();
+}
+
 // |ExternalViewEmbedder|
 void AndroidExternalViewEmbedder::BeginFrame(SkISize frame_size,
                                              GrContext* context,
                                              double device_pixel_ratio) {
+  Reset();
   frame_size_ = frame_size;
 }
 
-void AndroidExternalViewEmbedder::ClearFrame() {
-  composition_order_.clear();
-  picture_recorders_.clear();
-  frame_size_ = SkISize::MakeEmpty();
```

Comment:
Do we no longer need to set `frame_size` to empty anymore?

Reply:
No - `PrerollCompositeEmbeddedView` will always be preceded by a `BeginFrame` call that sets the `frame_size_`